ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -31,12 +31,47 @@
 
 
 def _get_fn_dtypes(framework, kind="valid"):
+    """
+    Return a list of supported data types for a specified framework and kind of data type.
```

Comment:
It is not only for the specified framework, this will return the supported data types for the current function being tested, we should add this to the description of the function and also to `kind` and `return` docstring.